solr/solrj-zookeeper/build.gradle

```diff
@@ -32,6 +32,13 @@ dependencies {
 
     implementation project(':solr:solrj')
 
+    api('org.apache.curator:curator-client', {
```

Comment:
Because there are multi-cmd methods that take Curator code as a parameter.

Reply:
Let me put it like this -- what if you set it to "implementation".  What bad things will happen?

Reply:
Actually implementation should be ok for the `curator-client`. For `curator-framework`, that wouldn't be great because of the `SolrZkClient.multi()` function parameters.